client/network_status.go

```diff
@@ -0,0 +1,154 @@
+package client
+
+import (
+	"fmt"
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/pkg/server"
+	"github.com/skupperproject/skupper/pkg/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (cli *VanClient) NetworkStatus() ([]*types.SiteInfo, error) {
+
+	//Checking if the router has been deployed
+	_, err := cli.KubeClient.AppsV1().Deployments(cli.Namespace).Get(types.TransportDeploymentName, metav1.GetOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("skupper is not installed: %s", err)
+	}
+
+	sites, err := server.GetSiteInfo(cli.Namespace, cli.KubeClient, cli.RestConfig)
+
+	if err != nil {
+		return nil, err
+	}
+
+	versionCheckedSites := cli.checkSiteVersion(sites)
+	siteNameMap := getSiteNameMap(sites)
+
+	services, err := server.GetServiceInfo(cli.Namespace, cli.KubeClient, cli.RestConfig)
+
+	if err != nil {
+		return nil, err
+	}
+
+	var listSites []*types.SiteInfo
+
+	for _, site := range versionCheckedSites {
+
+		if len(site.Namespace) == 0 {
+			return nil, fmt.Errorf("temporarily unable to provide site information")
```

Comment:
The gateway will show up in the site list without a namespace. Suggest that gateways be excluded from list (for now) and that site loop 'continue' here rather than return error.

If namespace check was also being used to detect incomplete site info, we should figure out a more deterministic way to identify.

Reply:
Thanks for finding this, gateways are filtered for now.